daemon/api.go

```diff
@@ -664,7 +664,8 @@ func searchStore(c *Command, r *http.Request, user *auth.UserState) Response {
 	}
 
 	theStore := getStore(c)
-	found, err := theStore.Find(&store.Search{
+	ctx := store.ClientUserAgentContext(r.Header.Get("User-Agent"))
```

Comment:
@mvo @chipaca I think ultimately we want the common layer that calls the handlers to create a Background context and this to operate on that, but that's  a bigger refactoring.